app/views/assignments/_form.html.erb

```diff
@@ -124,6 +150,16 @@
         .attr('value', JSON.stringify(updated_restrictions))
         .appendTo('#assignmentForm');
   }
+  function generateLtiKeys(){
+    let lti_credentials_needed = false;
+    if ($('#lms-integration-check').is(":checked")) {
+      lti_credentials_needed = true;
+    }
+    $('<input />').attr('type', 'hidden')
```

Comment:
Why not use https://apidock.com/rails/ActionView/Helpers/FormHelper/hidden_field directly ?

Reply:
in the ```restrict-elements``` section this same logic is applied, so I have implemented this like the same way such that understandability of the code remains same for the same type of applications , should I change this ?? 
Please let me know

Reply:
done